Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ruby:2.5.1
+
+ADD . /code
+
+WORKDIR /code
+
+RUN apt-get update
+RUN apt-get install -y nodejs
+
+RUN gem install bundler
+RUN bundler
```

Comment:
I typically run `bundle install`; while they do the same thing, `bundler` is a wrapper around `bundle` and it might be nice to be explicit with the install?

Reply:
Done =)